### DISCUSSION TITLE
Add support CMIB for velocity

### DISCUSSION THREAD

**Discussion by Snele** [Zrips/CMI#7230] [Enhancement]
CMIB does not work on the velocity, is it possible to make it work?

**Comment by CloudeLecaw** [comment]
no it is not, as there is no offical support for velocity.

sorry about.

**Comment by Fabrimat** [comment]
Wish to see that soon!

**Comment by mrfloris** [comment]
Suggestions are welcomed in the discussions/ section, this is issues/, which is reserved for bugs. 

There's already a discussion going for velocity support, feel free to contribute there.

**Comment by Fabrimat** [comment]
Now we have duplicated discussions, let's use #2515

**Comment by mrfloris** [comment]
https://www.spigotmc.org/resources/cmi-298-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/update?update=521168

There is a mention in the changelog for velocity support in today's CMI release.
You can get the free cmiv jar file here: https://www.zrips.net/cmiv/
